packages/manim/build.sh

```diff
@@ -7,7 +7,7 @@ TERMUX_PKG_SRCURL=https://github.com/ManimCommunity/manim/archive/refs/tags/v$TE
 TERMUX_PKG_SHA256=36ce308b92bbbe039ee8cf4fbad5a8ba0ec5f24c9cf2919fab0fc5df92be7655
 TERMUX_PKG_DEPENDS="ffmpeg, libcairo, pango, xorgproto, python-numpy, python-pillow, pycairo, python-scipy, python-skia-pathops"
 TERMUX_PKG_RECOMMENDS="texlive-installer"
-TERMUX_PKG_PYTHON_TARGET_DEPS="av, click, cloup, decorator, importlib-metadata, isosurfaces, 'manimpango<1.0.0', mapbox-earcut, moderngl, moderngl-window, networkx, pydub, Pygments, rich, screeninfo, srt, svgelements, tqdm, typing-extensions, watchdog"
+TERMUX_PKG_PYTHON_TARGET_DEPS="'av==12.3.0', click, cloup, decorator, importlib-metadata, isosurfaces, 'manimpango<1.0.0', mapbox-earcut, moderngl, moderngl-window, networkx, pydub, Pygments, rich, screeninfo, srt, svgelements, tqdm, typing-extensions, watchdog"
```

Comment:
Please add `TERMUX_PKG_REVISION=1` underneath the package version to signify a package revision.